ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
+
+# A file that contained extra auths to be included in the pull secret.
+disconnected_registry_auths_file:
+
+# Name of a file containing the additionalTrustBundle and mirrored
+#   repository settings. If this is set, then a local registry will not be created.
+#   Setting this causes the playbook to use an existing disconnected registry.
+disconnected_registry_mirrors_file:
+
+# The containers registry port
+registry_port_container: 5000
+
+# The port exposed on the server for the registry
+registry_port: "{{ registry_port_container }}"
+
+# The following variables define the registry location
+registry_dir: /opt/registry
+registry_dir_auth: "{{ registry_dir }}/auth"
+registry_dir_cert: "{{ registry_dir }}/certs"
+registry_dir_data: "{{ registry_dir }}/data"
+
+# Name of the pod running as the registry.
+pod_name_registry: ocpdiscon-registry
```

Comment:
belong in installer/vars/main.yml -- or you can just hardcode the name when you use the podman_container module

Reply:
i dont agree. placing in defaults allows the tasks to potentially create multiple registries on the same system.

Reply:
I don't follow. What I'm trying to say here with registry_dir_{auth,cert,data} and podman_name_registry is these aren't values that you necessarily want changed. Values that maybe changed should go in defaults/main.yml however for values that aren't to change we should place them in vars/main.yml of that role

Reply:
i can move this to the vars dir if you want. making multiple registries on the same host probably wont be done often if at all.

Reply:
moved this to vars
pushed